Packs/PrismaCloudCompute/Scripts/PCComputeImageComplianceIssuesButton/README.md

```diff
@@ -0,0 +1,47 @@
+This script runs the "prisma-cloud-compute-images-scan-list" command for a specific container id and returns details about its compliance issues, if found. If any compliance issues found, it will create a new tab in the layout called "Detailed Compliance Issues" showing the issues details.
```

Comment:
```suggestion
This script runs the "prisma-cloud-compute-images-scan-list" command for a specific container ID and returns details about its compliance issues, if found. If any compliance issues found, it will create a new tab in the layout called "Detailed Compliance Issues" showing the issues details.
```